Hardcore.lua

```diff
@@ -447,6 +470,28 @@ function Hardcore:COMBAT_LOG_EVENT_UNFILTERED(...)
 	end
 end
 
+function Hardcore:GUILD_ROSTER_UPDATE(...)
+	guild_roster_loading = false
+
+	-- Create a new dictionary of just online people every time roster is updated
+	guild_online = {}
+
+	-- Hardcore:Debug('guild roster update')
```

Comment:
If this comment is no longer needed, might as well remove it.

Reply:
I left in a ton of debugging things commented them out because they spam the chat log.

I figured with the network code, we could keep it incase it needed to be debugged. I saw that was the case with other parts of the Addon.